dist/scss/wds-variables/_global-navigation.scss

```diff
@@ -0,0 +1,3 @@
+$wds-global-navigation-background-color: $wds-color-white;
+$wds-global-navigation-height: 55px;
+$wds-global-navigation-logo-width: 230px;
```

Comment:
Why do we need this variable public?
I can't find it used in other places than global navigation styles in DS.